ktor-server/jvm/src/io/ktor/server/application/ApplicationCall.kt

```diff
@@ -19,22 +20,35 @@ public interface ApplicationCall {
     public val application: Application
 
     /**
-     * Client request
+     * Client request.
```

Comment:
I think all of these parameters could do with a link back to the actual docs for this type of parameter. i.e. there's no need to explain again what server response is for instance, but having a link to the server response would be useful.